mst/failover.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Failover
+excerpt: Learn how maintenance is automatically handled on Managed Service for TimescaleDB
+product: mst
+keywords: [maintenance, update, upgrade, failover]
```

Comment:
I would suggest `HA` and `replica` as additional keywords

Reply:
done!!!!